lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1010,6 +1010,11 @@ void HdVP2Mesh::Sync(
             for (size_t i = 0; i < numFaceVertexIndices; i++) {
                 const int sceneFaceVtxId = faceVertexIndices[i];
 
+                // Scene index is actually greater than anticipated, increase the buffer size.
+                if (sceneFaceVtxId >= _meshSharedData->_sceneToRenderingFaceVtxIds.size()) {
+                    _meshSharedData->_sceneToRenderingFaceVtxIds.resize(sceneFaceVtxId + 1, -1);
```

Comment:
Does this trigger a resize only on the first index i ?

Reply:
I can trigger a resize whenever the scene face vertex ID is too large.